### ISSUE TITLE
[TL] ERROR: file models/pose.npz doesn't exist.

### ISSUE THREAD

**Issue by zeppafi** [tensorlayer/openpose-plus#152]
Hi,

Thanks for the awesome project!

I cloned the project and typed commands:

pip3 install -r requirements.txt
pip3 install pycocotools
python3 train.py

I tried to run last command for couple of times, but the same error seems to reoccur (see below). 

I'm running this on Linux:
Linux version 4.15.0-38-generic (buildd@lcy01-amd64-023) (gcc version 7.3.0 (Ubuntu 7.3.0-16ubuntu3)) #41-Ubuntu SMP Wed Oct 10 10:59:38 UTC 2018 

~/openpose-plus$ python3 train.py
[TL]     annotations exists
[TL]     validating images exists
[TL]     training images exists
[TL]     testing images exists
[x] Get pose data from data/mscoco2017/train2017
loading annotations into memory...
Done (t=11.38s)
creating index...
index created!
Overall get 56599 valid pose images from data/mscoco2017/train2017 and data/mscoco2017/annotations/person_keypoints_train2017.json
data/mscoco2017/train2017 has 56599 images
[x] Get pose data from data/mscoco2017/val2017
loading annotations into memory...
Done (t=0.24s)
creating index...
index created!
Overall get 2346 valid pose images from data/mscoco2017/val2017 and data/mscoco2017/annotations/person_keypoints_val2017.json
data/mscoco2017/val2017 has 2346 images
[TL] InputLayer  model/input: (?, 368, 368, 3)
[TL] Conv2d model/conv1_1: n_filter: 64 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/conv1_2: n_filter: 64 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] MaxPool2d model/pool1: filter_size: (2, 2) strides: (2, 2) padding: SAME
[TL] Conv2d model/conv2_1: n_filter: 128 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/conv2_2: n_filter: 128 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] MaxPool2d model/pool2: filter_size: (2, 2) strides: (2, 2) padding: SAME
[TL] Conv2d model/conv3_1: n_filter: 256 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/conv3_2: n_filter: 256 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/conv3_3: n_filter: 256 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/conv3_4: n_filter: 256 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] MaxPool2d model/pool3: filter_size: (2, 2) strides: (2, 2) padding: SAME
[TL] Conv2d model/conv4_1: n_filter: 512 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/conv4_2: n_filter: 512 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/conv4_3: n_filter: 256 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/conv4_4: n_filter: 128 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage1/branch1/c1: n_filter: 128 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage1/branch1/c2: n_filter: 128 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage1/branch1/c3: n_filter: 128 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage1/branch1/c4: n_filter: 512 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage1/branch1/confs: n_filter: 19 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] Conv2d model/stage1/branch2/c1: n_filter: 128 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage1/branch2/c2: n_filter: 128 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage1/branch2/c3: n_filter: 128 filter_size: (3, 3) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage1/branch2/c4: n_filter: 512 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage1/branch2/pafs: n_filter: 38 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] ConcatLayer model/stage2/concat: axis: -1
[TL] Conv2d model/stage2/branch1/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch1/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch1/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch1/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch1/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch1/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage2/branch1/conf: n_filter: 19 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] Conv2d model/stage2/branch2/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch2/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch2/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch2/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch2/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage2/branch2/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage2/branch2/pafs: n_filter: 38 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] ConcatLayer model/stage3/concat: axis: -1
[TL] Conv2d model/stage3/branch1/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch1/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch1/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch1/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch1/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch1/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage3/branch1/conf: n_filter: 19 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] Conv2d model/stage3/branch2/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch2/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch2/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch2/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch2/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage3/branch2/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage3/branch2/pafs: n_filter: 38 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] ConcatLayer model/stage4/concat: axis: -1
[TL] Conv2d model/stage4/branch1/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch1/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch1/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch1/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch1/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch1/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage4/branch1/conf: n_filter: 19 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] Conv2d model/stage4/branch2/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch2/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch2/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch2/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch2/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage4/branch2/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage4/branch2/pafs: n_filter: 38 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] ConcatLayer model/stage5/concat: axis: -1
[TL] Conv2d model/stage5/branch1/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch1/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch1/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch1/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch1/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch1/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage5/branch1/conf: n_filter: 19 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] Conv2d model/stage5/branch2/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch2/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch2/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch2/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch2/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage5/branch2/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage5/branch2/pafs: n_filter: 38 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] ConcatLayer model/stage6/concat: axis: -1
[TL] Conv2d model/stage6/branch1/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch1/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch1/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch1/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch1/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch1/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage6/branch1/conf: n_filter: 19 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL] Conv2d model/stage6/branch2/c1: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch2/c2: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch2/c3: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch2/c4: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch2/c5: n_filter: 128 filter_size: (7, 7) strides: (1, 1) pad: SAME act: relu
[TL] Conv2d model/stage6/branch2/c6: n_filter: 128 filter_size: (1, 1) strides: (1, 1) pad: VALID act: relu
[TL] Conv2d model/stage6/branch2/pafs: n_filter: 38 filter_size: (1, 1) strides: (1, 1) pad: VALID act: No Activation
[TL]   [*] geting variables with kernel
[TL]   got   0: model/conv1_1/kernel:0   (3, 3, 3, 64)
[TL]   got   1: model/conv1_2/kernel:0   (3, 3, 64, 64)
[TL]   got   2: model/conv2_1/kernel:0   (3, 3, 64, 128)
[TL]   got   3: model/conv2_2/kernel:0   (3, 3, 128, 128)
[TL]   got   4: model/conv3_1/kernel:0   (3, 3, 128, 256)
[TL]   got   5: model/conv3_2/kernel:0   (3, 3, 256, 256)
[TL]   got   6: model/conv3_3/kernel:0   (3, 3, 256, 256)
[TL]   got   7: model/conv3_4/kernel:0   (3, 3, 256, 256)
[TL]   got   8: model/conv4_1/kernel:0   (3, 3, 256, 512)
[TL]   got   9: model/conv4_2/kernel:0   (3, 3, 512, 512)
[TL]   got  10: model/conv4_3/kernel:0   (3, 3, 512, 256)
[TL]   got  11: model/conv4_4/kernel:0   (3, 3, 256, 128)
[TL]   got  12: model/stage1/branch1/c1/kernel:0   (3, 3, 128, 128)
[TL]   got  13: model/stage1/branch1/c2/kernel:0   (3, 3, 128, 128)
[TL]   got  14: model/stage1/branch1/c3/kernel:0   (3, 3, 128, 128)
[TL]   got  15: model/stage1/branch1/c4/kernel:0   (1, 1, 128, 512)
[TL]   got  16: model/stage1/branch1/confs/kernel:0   (1, 1, 512, 19)
[TL]   got  17: model/stage1/branch2/c1/kernel:0   (3, 3, 128, 128)
[TL]   got  18: model/stage1/branch2/c2/kernel:0   (3, 3, 128, 128)
[TL]   got  19: model/stage1/branch2/c3/kernel:0   (3, 3, 128, 128)
[TL]   got  20: model/stage1/branch2/c4/kernel:0   (1, 1, 128, 512)
[TL]   got  21: model/stage1/branch2/pafs/kernel:0   (1, 1, 512, 38)
[TL]   got  22: model/stage2/branch1/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  23: model/stage2/branch1/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  24: model/stage2/branch1/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  25: model/stage2/branch1/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  26: model/stage2/branch1/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  27: model/stage2/branch1/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  28: model/stage2/branch1/conf/kernel:0   (1, 1, 128, 19)
[TL]   got  29: model/stage2/branch2/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  30: model/stage2/branch2/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  31: model/stage2/branch2/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  32: model/stage2/branch2/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  33: model/stage2/branch2/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  34: model/stage2/branch2/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  35: model/stage2/branch2/pafs/kernel:0   (1, 1, 128, 38)
[TL]   got  36: model/stage3/branch1/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  37: model/stage3/branch1/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  38: model/stage3/branch1/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  39: model/stage3/branch1/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  40: model/stage3/branch1/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  41: model/stage3/branch1/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  42: model/stage3/branch1/conf/kernel:0   (1, 1, 128, 19)
[TL]   got  43: model/stage3/branch2/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  44: model/stage3/branch2/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  45: model/stage3/branch2/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  46: model/stage3/branch2/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  47: model/stage3/branch2/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  48: model/stage3/branch2/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  49: model/stage3/branch2/pafs/kernel:0   (1, 1, 128, 38)
[TL]   got  50: model/stage4/branch1/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  51: model/stage4/branch1/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  52: model/stage4/branch1/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  53: model/stage4/branch1/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  54: model/stage4/branch1/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  55: model/stage4/branch1/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  56: model/stage4/branch1/conf/kernel:0   (1, 1, 128, 19)
[TL]   got  57: model/stage4/branch2/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  58: model/stage4/branch2/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  59: model/stage4/branch2/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  60: model/stage4/branch2/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  61: model/stage4/branch2/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  62: model/stage4/branch2/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  63: model/stage4/branch2/pafs/kernel:0   (1, 1, 128, 38)
[TL]   got  64: model/stage5/branch1/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  65: model/stage5/branch1/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  66: model/stage5/branch1/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  67: model/stage5/branch1/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  68: model/stage5/branch1/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  69: model/stage5/branch1/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  70: model/stage5/branch1/conf/kernel:0   (1, 1, 128, 19)
[TL]   got  71: model/stage5/branch2/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  72: model/stage5/branch2/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  73: model/stage5/branch2/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  74: model/stage5/branch2/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  75: model/stage5/branch2/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  76: model/stage5/branch2/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  77: model/stage5/branch2/pafs/kernel:0   (1, 1, 128, 38)
[TL]   got  78: model/stage6/branch1/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  79: model/stage6/branch1/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  80: model/stage6/branch1/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  81: model/stage6/branch1/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  82: model/stage6/branch1/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  83: model/stage6/branch1/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  84: model/stage6/branch1/conf/kernel:0   (1, 1, 128, 19)
[TL]   got  85: model/stage6/branch2/c1/kernel:0   (7, 7, 185, 128)
[TL]   got  86: model/stage6/branch2/c2/kernel:0   (7, 7, 128, 128)
[TL]   got  87: model/stage6/branch2/c3/kernel:0   (7, 7, 128, 128)
[TL]   got  88: model/stage6/branch2/c4/kernel:0   (7, 7, 128, 128)
[TL]   got  89: model/stage6/branch2/c5/kernel:0   (7, 7, 128, 128)
[TL]   got  90: model/stage6/branch2/c6/kernel:0   (1, 1, 128, 128)
[TL]   got  91: model/stage6/branch2/pafs/kernel:0   (1, 1, 128, 38)
Start - n_step: 600000 batch_size: 8 lr_init: 4e-05 lr_decay_every_step: 136106
[TL] ERROR: file models/pose.npz doesn't exist.
2018-11-06 15:19:34.013385: W tensorflow/core/framework/op_kernel.cc:1261] Invalid argument: TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'
Traceback (most recent call last):

  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/ops/script_ops.py", line 206, in __call__
    ret = func(*args)

  File "train.py", line 154, in _data_aug_fn
    mask_miss = tl.prepro.affine_transform_cv2(mask_miss, transform_matrix, border_mode='replicate')

TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'

2018-11-06 15:19:34.013388: W tensorflow/core/framework/op_kernel.cc:1261] Invalid argument: TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'
Traceback (most recent call last):

  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/ops/script_ops.py", line 206, in __call__
    ret = func(*args)

  File "train.py", line 154, in _data_aug_fn
    mask_miss = tl.prepro.affine_transform_cv2(mask_miss, transform_matrix, border_mode='replicate')

TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'

2018-11-06 15:19:34.013904: W tensorflow/core/framework/op_kernel.cc:1261] Invalid argument: TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'
Traceback (most recent call last):

  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/ops/script_ops.py", line 206, in __call__
    ret = func(*args)

  File "train.py", line 154, in _data_aug_fn
    mask_miss = tl.prepro.affine_transform_cv2(mask_miss, transform_matrix, border_mode='replicate')

TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'

2018-11-06 15:19:34.014117: W tensorflow/core/framework/op_kernel.cc:1261] Invalid argument: TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'
Traceback (most recent call last):

  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/ops/script_ops.py", line 206, in __call__
    ret = func(*args)

  File "train.py", line 154, in _data_aug_fn
    mask_miss = tl.prepro.affine_transform_cv2(mask_miss, transform_matrix, border_mode='replicate')

TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'

2018-11-06 15:19:34.030183: W tensorflow/core/framework/op_kernel.cc:1261] Invalid argument: TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'
Traceback (most recent call last):

  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/ops/script_ops.py", line 206, in __call__
    ret = func(*args)

  File "train.py", line 154, in _data_aug_fn
    mask_miss = tl.prepro.affine_transform_cv2(mask_miss, transform_matrix, border_mode='replicate')

TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'

2018-11-06 15:19:34.037101: W tensorflow/core/framework/op_kernel.cc:1261] Invalid argument: TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'
Traceback (most recent call last):

  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/ops/script_ops.py", line 206, in __call__
    ret = func(*args)

  File "train.py", line 154, in _data_aug_fn
    mask_miss = tl.prepro.affine_transform_cv2(mask_miss, transform_matrix, border_mode='replicate')

TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'

2018-11-06 15:19:34.037555: W tensorflow/core/framework/op_kernel.cc:1261] Invalid argument: TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'
Traceback (most recent call last):

  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/ops/script_ops.py", line 206, in __call__
    ret = func(*args)

  File "train.py", line 154, in _data_aug_fn
    mask_miss = tl.prepro.affine_transform_cv2(mask_miss, transform_matrix, border_mode='replicate')

TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'

Traceback (most recent call last):
  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1334, in _do_call
    return fn(*args)
  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1319, in _run_fn
    options, feed_dict, fetch_list, target_list, run_metadata)
  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1407, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.InvalidArgumentError: TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'
Traceback (most recent call last):

  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/ops/script_ops.py", line 206, in __call__
    ret = func(*args)

  File "train.py", line 154, in _data_aug_fn
    mask_miss = tl.prepro.affine_transform_cv2(mask_miss, transform_matrix, border_mode='replicate')

TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'

	 [[{{node PyFunc}} = PyFunc[Tin=[DT_FLOAT, DT_STRING], Tout=[DT_FLOAT, DT_FLOAT, DT_FLOAT], token="pyfunc_3"](convert_image, arg1)]]
	 [[{{node IteratorGetNext}} = IteratorGetNext[output_shapes=[[?,368,368,3], [?,46,46,57], [?,46,46,1]], output_types=[DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](OneShotIterator)]]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "train.py", line 471, in <module>
    single_train(dataset)
  File "train.py", line 255, in single_train
    sess.run([train_op, total_loss, stage_losses, l2_loss, last_conf, last_paf])
  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 929, in run
    run_metadata_ptr)
  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1152, in _run
    feed_dict_tensor, options, run_metadata)
  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1328, in _do_run
    run_metadata)
  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/client/session.py", line 1348, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InvalidArgumentError: TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'
Traceback (most recent call last):

  File "/home/jaakko/.local/lib/python3.6/site-packages/tensorflow/python/ops/script_ops.py", line 206, in __call__
    ret = func(*args)

  File "train.py", line 154, in _data_aug_fn
    mask_miss = tl.prepro.affine_transform_cv2(mask_miss, transform_matrix, border_mode='replicate')

TypeError: affine_transform_cv2() got an unexpected keyword argument 'border_mode'

	 [[{{node PyFunc}} = PyFunc[Tin=[DT_FLOAT, DT_STRING], Tout=[DT_FLOAT, DT_FLOAT, DT_FLOAT], token="pyfunc_3"](convert_image, arg1)]]
	 [[node IteratorGetNext (defined at train.py:212)  = IteratorGetNext[output_shapes=[[?,368,368,3], [?,46,46,57], [?,46,46,1]], output_types=[DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](OneShotIterator)]]

**Comment by Jerry3062** [comment]
I met the same problem yet.

**Comment by luomai** [comment]
You would need to install the TensorLayer on the master branch as the augmentation is using a new API that is not available on the stable release.

**Comment by zeppafi** [comment]
Thanks, I was able to proceed. 

I had to run `pip3 uninstall tensorlayer`. After `pip3 show tensorlayer` didn't show anything, I typed `pip3 install https://github.com/tensorlayer/tensorlayer/archive/master.zip` (as documented on tensorlayer readme) and tested again with `python3 train.py`.

Now script proceeds to training phase even thought "[TL] ERROR: file models/pose.npz doesn't exist." is still printed.
